modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseSourceCheck.java

```diff
@@ -23,6 +23,12 @@
 import com.liferay.source.formatter.SourceFormatterMessage;
 import com.liferay.source.formatter.check.util.JSPSourceUtil;
 import com.liferay.source.formatter.check.util.SourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
```

Comment:
@TamyrisTorres, in the commit message it's missing an `e` in the word `includ`. It should be written as `include`. Also, remind yourself of https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752578/Wordsmith+conventions+in+Liferay#[inlineExtension]Human-readable-names,-not-Java-names, e.g. focus on human readability, therefore it should be `java class` instead of `javaClass`. Lastly, the words `Method` and `Java Variable` shouldn't have capital letters.